pubsub/hub.go

```diff
@@ -27,13 +27,16 @@ type hub struct {
 }
 
 func (h *hub) Subscribe() *Subscription {
-	id := uuid.NewV4()
-	subs := &Subscription{ID: id, Events: make(EventChan, 1000), OnClose: func(subID uuid.UUID) {
-		h.subscriptions.Delete(subID)
-	}}
-	h.subscriptions.Store(id, subs)
-
-	return subs
+	id, error := uuid.NewV4()
```

Comment:
Can we name the `error` variable as `err` ?

Reply:
yes, fixed

Reply:
@vmysak Did not get why did you need to handle this error ?